examples/cndpfwd/parse-args.c

```diff
@@ -301,6 +304,58 @@ print_usage(char *prog_name)
                prog_name, BURST_SIZE, MAX_BURST_SIZE, OPT_NO_COLOR);
 }
 
+static int
+_cleanup(jcfg_info_t *j __cne_unused, void *obj, void *arg, int idx __cne_unused)
+{
+    jcfg_thd_t *thd = obj;
+    jcfg_lport_t *lport;
+    struct fwd_info *fwd = arg;
+
+    foreach_thd_lport (thd, lport) {
+        if (lport->umem) {
+            for (int i = 0; i < lport->umem->region_cnt; i++) {
+                pktmbuf_destroy(lport->umem->rinfo[i].pool);
+                lport->umem->rinfo[i].pool = NULL;
+            }
+            mmap_free(lport->umem->mm);
+            lport->umem->mm = NULL; /* Make sure we do not free this again */
+        }
+
+        struct fwd_port *pd = lport->priv_;
+
+        if (!pd) {
```

Comment:
Remove single line {} for if, for, while, do, ...

Reply:
Updated